flask/test/model_tests/test_base.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from boxwise_flask.models.base import Base
+from playhouse.shortcuts import model_to_dict
+
+
+@pytest.mark.usefixtures("default_base")
+def test_get_base_from_id(default_base):
+    base = Base.get_from_id(default_base["id"])
+    base_dict = model_to_dict(base)
```

Comment:
You could use `base.id`, `base.seq` etc instead of converting to a dict. I find it'd highlight the difference between expected (dict) and actual (model instance) content